lang/cs/Org.Apache.REEF.Network/Group/Config/GroupCommConfigurationOptions.cs

```diff
@@ -44,7 +44,7 @@ public class Timeout : Name<int>
         {
         }
 
-        [NamedParameter("Retry times", defaultValue: "5")]
+        [NamedParameter("Retry times", defaultValue: "500")]
         public class RetryCount : Name<int>
```

Comment:
All the more reason to resolve this now. Maybe this number needs to be provided by an injectable interface? That way, it could be computed at runtime.